docs/conf.py

```diff
@@ -20,72 +14,67 @@
-# ones.
-extensions = ["myst_parser", "sphinx_copybutton"]
+project = 'docker-stacks'
+copyright = '2022, Project Jupyter'
+author = 'Project Jupyter'
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
+# The short X.Y version
+version = 'latest'
 
-source_suffix = {
-    ".rst": "restructuredtext",
-    ".md": "markdown",
-}
+# The full version, including alpha/beta/rc tags
+release = 'latest'
 
-# The master toctree document.
-master_doc = "index"
 
-# General information about the project.
-project = "docker-stacks"
-copyright = "2018- Project Jupyter"
-author = "Project Jupyter"
+# -- General configuration ---------------------------------------------------
 
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-version = "latest"
-# The full version, including alpha/beta/rc tags.
-release = "latest"
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-# This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 
-# -- Options for HTML output ----------------------------------------------
+# -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "alabaster"
+html_theme = 'alabaster'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
+
+# File above was generated using this command:
+# sphinx-quickstart --project "docker-stacks" --author "Project Jupyter" -v "latest" -r "latest" -l en --no-sep --no-makefile --no-batchfile
```

Comment:
I generated the file above using this command line and only appended settings to it.
This will make it easy to update to newer sphinx versions and keeps this file clean.

Reply:
I love when these kinds of notes are added, nice!

You left out though what version of sphinx was used, was it sphinx 4?

Reply:
It was the latest one, 4.3.2.
I will add version here.